### ISSUE TITLE
[Build 2024110718-4.3] openQA test fails in clipboard_and_web with sys-gui

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#9564]
## Observation

openQA test in scenario qubesos-4.3-pull-requests-x86_64-system_tests_guivm_gui_interactive@64bit fails in
[clipboard_and_web](https://openqa.qubes-os.org/tests/117927/modules/clipboard_and_web/steps/11)

Copy notification does not appear. Logs suggest copy didn't happened at all.

## Reproducible

Fails since (at least) Build [2024110603-4.3](https://openqa.qubes-os.org/tests/117639)

## Expected result

Last good: [2024110421-4.3](https://openqa.qubes-os.org/tests/117542) (or more recent)

## Further details

Always latest result in this scenario: [latest](https://openqa.qubes-os.org/tests/latest?arch=x86_64&distri=qubesos&flavor=pull-requests&machine=64bit&test=system_tests_guivm_gui_interactive&version=4.3)

**Comment by marmarek** [comment]
Ah, I know, gui-daemon in dom0 intercepted the copy key sequence. This is because it didn't got option to ignore it:
```
qvm-start-daemon: sys-gui: Ignoring invalid feature:
gui-secure-copy-sequence=none
qvm-start-daemon: sys-gui: Ignoring invalid feature:
gui-secure-paste-sequence=none
```

**Comment by alimirjamali** [comment]
> Ah, I know, gui-daemon in dom0 intercepted the copy key sequence. This is because it didn't got option to ignore it:

I see. The `parse_key_sequence` procedure within `xside.c` does not exit on invalid key sequences and will be just disable it (also related to my recent closed issue #9561).
